scripts/python/lib/validate_config_logic.py

```diff
@@ -211,8 +211,8 @@ def _validate_deployer_networks(self):
         netprefix = self.cfg.get_depl_netw_mgmt_prefix()
         cont_ip = self.cfg.get_depl_netw_mgmt_cont_ip()
         br_ip = self.cfg.get_depl_netw_mgmt_brg_ip()
-        if cont_ip[0]:
-            for i, cip in enumerate(cont_ip):
+        for i, cip in enumerate(cont_ip):
+            if cip:
```

Comment:
I don't think this 'if' statement is necessary.

Reply:
The (switch) management network can be created with or without a tagged vlan and we actually allow the user to create both types together. For instance, the config file content below is currently allowed in the config file.  In this case the first management network has no container ip address and no bridge ip address. The if statement only verifies that the container and bridge ip are in the same subnet if there is a container ip address in the defined network.
deployer:
    gateway: true
    networks:
        mgmt:
            - device: enp1s0f1
              interface_ipaddr: 192.168.33.99
              netmask: 255.255.255.0
            - device: enp1s0f1
              container_ipaddr: 192.168.16.2
              bridge_ipaddr: 192.168.16.3
              vlan: 16
              prefix: 24

Reply:
Thanks for the explanation.